tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -55,7 +56,9 @@ SCHEDULER_PORT=8786
 DASHBOARD_ADDRESS=8787
 
 if [ "$ROLE" = "SCHEDULER" ]; then
+  echo "I'm scheduler!"
   if [ "$CLUSTER_MODE" = "NVLINK" ]; then
+     echo "Starting sched.."
```

Comment:
We should add this for both `NVLINK` and `TCP` with something like:
```suggestion
     echo "Starting NVLINK based Scheduler"
```